sdk/zklogin/src/address.ts

```diff
@@ -49,8 +49,11 @@ export function computeZkAddress({
 	aud,
 	userSalt,
 }: ComputeZKAddressOptions) {
-	const addressSeedBytesBigEndian = toBufferBE(genAddressSeed(userSalt, claimName, claimValue), 32);
-	const addressParamBytes = zkBcs.ser('AddressParams', { iss, aud }).toBytes();
+	const addressSeedBytesBigEndian = toBufferBE(
+		genAddressSeed(userSalt, claimName, claimValue, aud),
+		32,
+	);
+	const addressParamBytes = zkBcs.ser('AddressParams', { iss }).toBytes();
```

Comment:
We could update the bcs struct if we don't expect to have the aud any more

Reply:
I updated this [here](https://github.com/MystenLabs/sui/pull/13648/files#diff-fb78cfe51f430014f14d56cb22fde0a645e74dcd49f08091839e3c1a2806e002L12) because it fails without the aud